CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Mapbox welcomes participation and contributions from everyone.
 ## Unreleased
 #### Features
 #### Bug fixes and improvements
+- Fixed `BannerInstructions` issue: the banner instruction might be removed from `RouteProgress` at some point around a edge's leg. [#6684](https://github.com/mapbox/mapbox-navigation-android/pull/6684)
```

Comment:
Fixed `BannerInstructions` issue where the banner instruction might have been removed

Reply:
I'd highlight that instructions are being removed around the legs` edges

Reply:
I meant:
"- Fixed `BannerInstructions` issue where the banner instruction might have been removed from `RouteProgress` at some point around a edge's leg. [#6684](https://github.com/mapbox/mapbox-navigation-android/pull/6684)"
:)
It's just when it's in present it's not clear if that was the issue or the current behaviour.

Reply:
ok, have nothing against it. addressed in https://github.com/mapbox/mapbox-navigation-android/pull/6688